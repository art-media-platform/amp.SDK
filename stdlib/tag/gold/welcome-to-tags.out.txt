                                    amp.spec.tag.v.0.8  . ✙ בְּרֵאשִׁ֖ית בָּרָ֣א אֱלֹהִ֑ים אֵ֥ת הַשָּׁמַ֖יִם וְאֵ֥ת הָאָֽרֶץ  ✙ .
9FDP7NU5U8N2PQU8BHSNUF82B77BBZUQE   amp.spec.tag.v.0.8.✙.בְּרֵאשִׁ֖ית.בָּרָ֣א.אֱלֹהִ֑ים.אֵ֥ת.הַשָּׁמַ֖יִם.וְאֵ֥ת.הָאָֽרֶץ.✙

                                    Hello World!
 B3EJD9HGSN053Z2TWV6920UP70HDZPNP   hello.world
                                    Hello, World!
 B3EJD9HGSN053Z2TWV6920UP70HDZPNP   hello.world
                                    hello  world
 B3EJD9HGSN053Z2TWV6920UP70HDZPNP   hello.world
                                    world  hello
 B3EJD9HGSN053Z2TWV6920UP70HDZPNP   world.hello
                                    World, hello.
 B3EJD9HGSN053Z2TWV6920UP70HDZPNP   world.hello

                                    Hello and welcome to tags!
39ZRS85V2W78NZ5NE13YWFHUHJSKRRX3R   hello.and.welcome.to.tags
                                    hello, and welcome to tags!
39ZRS85V2W78NZ5NE13YWFHUHJSKRRX3R   hello.and.welcome.to.tags
                                    hello, and Welcome to tags!!
39ZRS85V2W78NZ5NE13YWFHUHJSKRRX3R   hello.and.welcome.to.tags
                                    hello, and..welcome to... tags.
39ZRS85V2W78NZ5NE13YWFHUHJSKRRX3R   hello.and.welcome.to.tags
                                    hello+and+welcome+to+tags
39ZRS85V2W78NZ5NE13YWFHUHJSKRRX3R   hello.and.welcome.to.tags


                                    UTF8 whitespace, commas, +, and . and : denote the "with" operator.   This operator adds the hash of each operand and is also therefore commutative.
9J5QZPJ2XGSNR0NHH7UF72J36FFD5N46G   UTF8.whitespace.commas.and.and.denote.the."with".operator.this.operator.adds.the.hash.of.each.operand.and.is.also.therefore.commutative

                                    Text having "homophone case" or "homophonic case" means its component words are considered interchangeable with words that sound the same but have different spelling.
BNVBK7822R57ZDWVFBT5BM82FTSMP02VD   text.having."homophone.case".or."homophonic.case".means.its.component.words.are.considered.interchangeable.with.words.that.sound.the.same.but.have.different.spelling

                                    The words Center, center, centre, and Centre are homophonic.
4PBUM8VHVRVN9H8F4Q9NP9HRJZ5UY5Z9Z   the.words.center.center.centre.and.centre.are.homophonic
                                    The words center, Center, Centre, and centre are homophonic.
4PBUM8VHVRVN9H8F4Q9NP9HRJZ5UY5Z9Z   the.words.center.center.centre.and.centre.are.homophonic
                                    The words Centre, centre, center, and Center are homophonic.
4PBUM8VHVRVN9H8F4Q9NP9HRJZ5UY5Z9Z   the.words.centre.centre.center.and.center.are.homophonic

                                    This has very helpful implications for those with sight or hearing impairments, or in search result ranking since the query word order is usually less important the presence of particular words, especially once there are dozen words or more.
HNGM76C66X9WYF9G32P0G5T2ZMZ52WHWE   this.has.very.helpful.implications.for.those.with.sight.or.hearing.impairments.or.in.search.result.ranking.since.the.query.word.order.is.usually.less.important.the.presence.of.particular.words.especially.once.there.are.dozen.words.or.more

                                    However: dashes and slashes are order dependent!?  Yes, order-dependent!
36711JP6E0TBK2Z84NS5WJ8NNKEWB7G3P   however.dashes.and.slashes.are.order.dependent.yes.order-dependent
                                    However, dashes and slashes are order dependent?!  yes, order-dependent
36711JP6E0TBK2Z84NS5WJ8NNKEWB7G3P   however.dashes.and.slashes.are.order.dependent.yes.order-dependent
                                    However, Slashes and dashes are order dependent?  Yes, order-dependent
36711JP6E0TBK2Z84NS5WJ8NNKEWB7G3P   however.slashes.and.dashes.are.order.dependent.yes.order-dependent
                                    However: Order-dependent?  Yes, slashes, and dashes are order dependent.
36711JP6E0TBK2Z84NS5WJ8NNKEWB7G3P   however.order-dependent.yes.slashes.and.dashes.are.order.dependent
                                    however: dashes and slashes are order-dependent! yES! Order dependent!
36711JP6E0TBK2Z84NS5WJ8NNKEWB7G3P   however.dashes.and.slashes.are.order-dependent.yes.order.dependent

                                    However: dependent-order?  Yes, slashes and dashes are order dependent.
4FCMDX3008CJD7PD9QJZZQT80X82NTVYC   however.dependent-order.yes.slashes.and.dashes.are.order.dependent
                                    However: Slashes and dashes are order dependent. Yes, dependent-order?
4FCMDX3008CJD7PD9QJZZQT80X82NTVYC   however.slashes.and.dashes.are.order.dependent.yes.dependent-order
                                    -order dependent, yes. Slashes and dashes are order dependent.  However.
4FCMDX3008CJD7PD9QJZZQT80X82NTVYC   -order.dependent.yes.slashes.and.dashes.are.order.dependent.however

                                    Acronyms like NBA, NFL, USA, CIA, PRC, and EU are capitalized since they are spoken letter by letter.
713VSQXU2V4JVRQQG2DMGNFS0CDRBDN4K   acronyms.like.NBA.NFL.USA.CIA.PRC.and.EU.are.capitalized.since.they.are.spoken.letter.by.letter

                                    Words that are not upper case, are considered spoken / phonetic.
1Y3K9HH71TK5P049NG42QJ3MBD9RDYKXN   words.that.are.not.upper.case.are.considered.spoken-phonetic

                                    This also means we can communicate proper nouns and titles easily, such as A Game Of Thrones
9X1QZ2F0P8ET35Q15H94BS0RN83Q4J050   this.also.means.we.can.communicate.proper.nouns.and.titles.easily.such.as.a.game.of.thrones
                                    This also means we can communicate proper nouns and titles easily, such as Game of Thrones, A.
9X1QZ2F0P8ET35Q15H94BS0RN83Q4J050   this.also.means.we.can.communicate.proper.nouns.and.titles.easily.such.as.game.of.thrones.a
                                    This also means we can communicate proper nouns and titles easily, such as a game of thrones.
9X1QZ2F0P8ET35Q15H94BS0RN83Q4J050   this.also.means.we.can.communicate.proper.nouns.and.titles.easily.such.as.a.game.of.thrones

                                    Notice how acronyms are spoken differently and have their own unique meanings, so that means ALL-CAPS should be used intentionally.
9NWT2RZYNHUQQQZUEX71T04N9MJZ9NTNK   notice.how.acronyms.are.spoken.differently.and.have.their.own.unique.meanings.so.that.means.ALL-CAPS.should.be.used.intentionally

                                    ADA is an acronym for the Americans with Disabilities Act, Ms. Ada is a computer programmer.
7RBXE95STWGNE346S007PGSU2KZQZ2Y1E   ADA.is.an.acronym.for.the.americans.with.disabilities.act.ms.ada.is.a.computer.programmer
                                    ADA is an acronym for the Americans with Disabilities Act. ms ada is a computer programmer!
7RBXE95STWGNE346S007PGSU2KZQZ2Y1E   ADA.is.an.acronym.for.the.americans.with.disabilities.act.ms.ada.is.a.computer.programmer
                                    ADA is an acronym for the Americans with Disabilities Act. Ada is a computer programmer, Ms!
7RBXE95STWGNE346S007PGSU2KZQZ2Y1E   ADA.is.an.acronym.for.the.americans.with.disabilities.act.ada.is.a.computer.programmer.ms
                                    ADA is an acronym for the Americans With Disabilities Act, A computer programmer, Ms. ada is.
7RBXE95STWGNE346S007PGSU2KZQZ2Y1E   ADA.is.an.acronym.for.the.americans.with.disabilities.act.a.computer.programmer.ms.ada.is

                                    Get into the amp SDK today!
3RZ1NT53UHXHU9DQXW40QW90WTEJ39HFM   get.into.the.amp.SDK.today
                                    Get into the amp.SDK today!
3RZ1NT53UHXHU9DQXW40QW90WTEJ39HFM   get.into.the.amp.SDK.today
                                    Get into the Amp SDK today!
3RZ1NT53UHXHU9DQXW40QW90WTEJ39HFM   get.into.the.amp.SDK.today


                                    Yoda said this is not a drill.
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   yoda.said.this.is.not.a.drill
                                    Yoda said, a drill this is not.
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   yoda.said.a.drill.this.is.not
                                    Yoda said, a drill this is not!
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   yoda.said.a.drill.this.is.not
                                    a drill this is not, Yoda said!
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   a.drill.this.is.not.yoda.said
                                    said a drill, this is not Yoda.
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   said.a.drill.this.is.not.yoda
                                    said a drill, is this not Yoda?
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   said.a.drill.is.this.not.yoda
                                    said a drill: Is this not Yoda?
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   said.a.drill.is.this.not.yoda
                                    Said a drill,: Is this not Yoda??
2HKM9422MHYS0Z36YM9BH9J48W7Q0YS8G   said.a.drill.is.this.not.yoda

                                    12 words randomly chosen from, but not limited to, the English dictionary exceeds 10^56 possible combinations.  For reference, that's over 10 million times the number of possible Bitcoin public addresses (2^160).
G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB   12.words.randomly.chosen.from.but.not.limited.to.the.english.dictionary.exceeds.10-56.possible.combinations.for.reference.that's.over.10.million.times.the.number.of.possible.bitcoin.public.addresses.(2-160)

                                    The above hash ID G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB demonstrates a standardized way to precisely witness a set of unordered literals. In creating a subsequent hash ID, we are cryptographically chaining tag sets together.
BSJ6041BGWJM08K4S0M05995JZSQRHPV2   the.above.hash.ID.G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB.demonstrates.a.standardized.way.to.precisely.witness.a.set.of.unordered.literals.in.creating.a.subsequent.hash.ID.we.are.cryptographically.chaining.tag.sets.together

                                    And so by citing both G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB and BSJ6041BGWJM08K4S0M05995JZSQRHPV2 as literals, we can conveniently validate tag relationships and groups.
9X75KTQ662RMTKRTNF0FY2PSWUC4N21G3   and.so.by.citing.both.G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB.and.BSJ6041BGWJM08K4S0M05995JZSQRHPV2.as.literals.we.can.conveniently.validate.tag.relationships.and.groups


