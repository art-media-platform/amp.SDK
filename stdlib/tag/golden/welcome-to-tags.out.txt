                               amp.spec.tag.v0.703  . ✙ בְּרֵאשִׁ֖ית בָּרָ֣א אֱלֹהִ֑ים אֵ֥ת הַשָּׁמַ֖יִם וְאֵ֥ת הָאָֽרֶץ  ✙ .
  5UEB5DMHWE0FUK5SXW85BRD05F   amp.spec.tag.v0.703.✙.בְּרֵאשִׁ֖ית.בָּרָ֣א.אֱלֹהִ֑ים.אֵ֥ת.הַשָּׁמַ֖יִם.וְאֵ֥ת.הָאָֽרֶץ.✙

                               Hello World!
  172RE53M76C7Y540651UBXEN10   hello.world
                               Hello, World!
  172RE53M76C7Y540651UBXEN10   hello.world
                               hello  world
  172RE53M76C7Y540651UBXEN10   hello.world
                               world  hello
  172RE53M76C7Y540651UBXEN10   world.hello
                               World, hello.
  172RE53M76C7Y540651UBXEN10   world.hello

                               Hello and welcome to tags!
   UHGRNY009JFCP1N70DEWZ4Z14   hello.and.welcome.to.tags
                               hello, and welcome to tags!
   UHGRNY009JFCP1N70DEWZ4Z14   hello.and.welcome.to.tags
                               hello, and Welcome to tags!!
   UHGRNY009JFCP1N70DEWZ4Z14   hello.and.welcome.to.tags
                               hello, and..welcome to... tags.
   UHGRNY009JFCP1N70DEWZ4Z14   hello.and.welcome.to.tags
                               hello+and+welcome+to+tags
   UHGRNY009JFCP1N70DEWZ4Z14   hello.and.welcome.to.tags


                               UTF8 whitespace, commas, +, and . denote the "with" operator.   This operator adds the hash of each operand and is also therefore commutative.
  7ZZWEFBYWDUDQQ9F9B94GK3WEJ   UTF8.whitespace.commas.and.denote.the."with".operator.this.operator.adds.the.hash.of.each.operand.and.is.also.therefore.commutative

                               Text having "homophone case" or "homophonic case" means its component words are considered interchangeable with words that sound the same but have different spelling.
  5C43BBNPBMETTZHMXU85V0YN8U   text.having."homophone.case".or."homophonic.case".means.its.component.words.are.considered.interchangeable.with.words.that.sound.the.same.but.have.different.spelling

                               The words Center, center, centre, and Centre are homophonic.
  3N6KTUQV5KSDGTMUEY3J1ZTSVJ   the.words.center.center.centre.and.centre.are.homophonic
                               The words center, Center, Centre, and centre are homophonic.
  3N6KTUQV5KSDGTMUEY3J1ZTSVJ   the.words.center.center.centre.and.centre.are.homophonic
                               The words Centre, centre, center, and Center are homophonic.
  3N6KTUQV5KSDGTMUEY3J1ZTSVJ   the.words.centre.centre.center.and.center.are.homophonic

                               This has very helpful implications for those with sight or hearing impairments, or in search result ranking since the query word order is usually less important the presence of particular words, especially once there are dozen words or more.
  20FGJB8NSMWDMFHXZBT6H4HCPK   this.has.very.helpful.implications.for.those.with.sight.or.hearing.impairments.or.in.search.result.ranking.since.the.query.word.order.is.usually.less.important.the.presence.of.particular.words.especially.once.there.are.dozen.words.or.more

                               However, colons, dashes and slashes are order dependent!?  Yes, order-dependent!
  4QHWPP5B42Y8M5SRUB8G60S6NW   however.colons.dashes.and.slashes.are.order.dependent.yes.order-dependent
                               However, colons, dashes and slashes are order dependent?!  yes, order-dependent
  4QHWPP5B42Y8M5SRUB8G60S6NW   however.colons.dashes.and.slashes.are.order.dependent.yes.order-dependent
                               However, Slashes, colons and dashes are order dependent?  Yes, order-dependent
  4QHWPP5B42Y8M5SRUB8G60S6NW   however.slashes.colons.and.dashes.are.order.dependent.yes.order-dependent
                               However, Order-dependent?  Yes, colons, slashes, and dashes are order dependent.
  4QHWPP5B42Y8M5SRUB8G60S6NW   however.order-dependent.yes.colons.slashes.and.dashes.are.order.dependent
                               however, dashes, colons and slashes are order-dependent! yES! Order dependent!
  4QHWPP5B42Y8M5SRUB8G60S6NW   however.dashes.colons.and.slashes.are.order-dependent.yes.order.dependent

                               Acronyms like NBA, NFL, USA, CIA, PRC, and EU are capitalized since they are spoken letter by letter.
  43JCT0KD0KTMEVU8YS86EP59T5   acronyms.like.NBA.NFL.USA.CIA.PRC.and.EU.are.capitalized.since.they.are.spoken.letter.by.letter

                               Words that are not upper case, are considered spoken / phonetic.
  23KVJG22MV6H7NX36FVUV4VMVC   words.that.are.not.upper.case.are.considered.spoken-phonetic

                               This also means we can communicate proper nouns and titles easily, such as A Game Of Thrones
  1ZY1RYHN444MC34WK32YNS0JR3   this.also.means.we.can.communicate.proper.nouns.and.titles.easily.such.as.a.game.of.thrones
                               This also means we can communicate proper nouns and titles easily, such as Game of Thrones, A.
  1ZY1RYHN444MC34WK32YNS0JR3   this.also.means.we.can.communicate.proper.nouns.and.titles.easily.such.as.game.of.thrones.a
                               This also means we can communicate proper nouns and titles easily, such as a game of thrones.
  1ZY1RYHN444MC34WK32YNS0JR3   this.also.means.we.can.communicate.proper.nouns.and.titles.easily.such.as.a.game.of.thrones

                               Notice how acronyms are spoken differently and have their own unique meanings, so that means ALL-CAPS should be used intentionally.
   858TBGKRB9TYQVCN4UKF6G63N   notice.how.acronyms.are.spoken.differently.and.have.their.own.unique.meanings.so.that.means.ALL-CAPS.should.be.used.intentionally

                               ADA is an acronym for the Americans with Disabilities Act, Ms. Ada is a computer programmer.
  51UFVVK57XUY8X7UG43DV7JVZ5   ADA.is.an.acronym.for.the.americans.with.disabilities.act.ms.ada.is.a.computer.programmer
                               ADA is an acronym for the Americans with Disabilities Act. ms ada is a computer programmer!
  51UFVVK57XUY8X7UG43DV7JVZ5   ADA.is.an.acronym.for.the.americans.with.disabilities.act.ms.ada.is.a.computer.programmer
                               ADA is an acronym for the Americans with Disabilities Act. Ada is a computer programmer, Ms!
  51UFVVK57XUY8X7UG43DV7JVZ5   ADA.is.an.acronym.for.the.americans.with.disabilities.act.ada.is.a.computer.programmer.ms
                               ADA is an acronym for the Americans With Disabilities Act, A computer programmer, Ms. ada is.
  51UFVVK57XUY8X7UG43DV7JVZ5   ADA.is.an.acronym.for.the.americans.with.disabilities.act.a.computer.programmer.ms.ada.is

                               Get into the amp SDK today!
  4R9R1VY0Z1VQSB8SD7GGBW4EKZ   get.into.the.amp.SDK.today
                               Get into the amp.SDK today!
  4R9R1VY0Z1VQSB8SD7GGBW4EKZ   get.into.the.amp.SDK.today
                               Get into the Amp SDK today!
  4R9R1VY0Z1VQSB8SD7GGBW4EKZ   get.into.the.amp.SDK.today


                               Yoda said this is not a drill.
   DK2K88Q4VVU60ZCW8BN9Q23R8   yoda.said.this.is.not.a.drill
                               Yoda said, a drill this is not.
   DK2K88Q4VVU60ZCW8BN9Q23R8   yoda.said.a.drill.this.is.not
                               Yoda said, a drill this is not!
   DK2K88Q4VVU60ZCW8BN9Q23R8   yoda.said.a.drill.this.is.not
                               a drill this is not, Yoda said!
   DK2K88Q4VVU60ZCW8BN9Q23R8   a.drill.this.is.not.yoda.said
                               said a drill, this is not Yoda.
   DK2K88Q4VVU60ZCW8BN9Q23R8   said.a.drill.this.is.not.yoda
                               said a drill, is this not Yoda?
   DK2K88Q4VVU60ZCW8BN9Q23R8   said.a.drill.is.this.not.yoda

                               12 words randomly chosen from, but not limited to, the English dictionary exceeds 10^56 possible combinations.  For reference, that's over 10 million times the number of possible Bitcoin public addresses (2^160).
   CPDS2JN244BEZCS8SRFWM71FC   12.words.randomly.chosen.from.but.not.limited.to.the.english.dictionary.exceeds.10-56.possible.combinations.for.reference.that's.over.10.million.times.the.number.of.possible.bitcoin.public.addresses.(2-160)

                               The above hash ID G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB demonstrates a standardized way to precisely witness a set of unordered literals. In creating a subsequent hash ID, we are cryptographically chaining tag sets together.
   Q0P2W7N84KX4H0S7BBET5HXSW   the.above.hash.ID.G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB.demonstrates.a.standardized.way.to.precisely.witness.a.set.of.unordered.literals.in.creating.a.subsequent.hash.ID.we.are.cryptographically.chaining.tag.sets.together

                               And so by citing both G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB and BSJ6041BGWJM08K4S0M05995JZSQRHPV2 as literals, we can conveniently validate tag relationships and groups.
  6Z536YQ9MJ6EG9JU8NSRQH4GY3   and.so.by.citing.both.G5NRUJZEX37FZE7J38C0PRSQNCGF8DCXB.and.BSJ6041BGWJM08K4S0M05995JZSQRHPV2.as.literals.we.can.conveniently.validate.tag.relationships.and.groups

                               Hashing 9X75KTQ662RMTKRTNF0FY2PSWUC4N21G3 with H3 geo-tiles 8D489E34600637F and 872AA845AFFFFFF demonstrates how locations can be cryptographically bound together in this system.  This offers easy "DIY" layering of security and permissions.
   N5BBQ9P2SS0VJEBJ0B6Z942FH   hashing.9X75KTQ662RMTKRTNF0FY2PSWUC4N21G3.with.H3.geo-tiles.8D489E34600637F.and.872AA845AFFFFFF.demonstrates.how.locations.can.be.cryptographically.bound.together.in.this.system.this.offers.easy."DIY".layering.of.security.and.permissions
